src/Sylius/Behat/Context/Transform/CartContext.php

```diff
@@ -57,7 +67,7 @@ public function providePreviousCartToken(): ?string
     public function provideLatestCart(): OrderInterface
     {
         $carts = $this->orderRepository->findBy(
-            ['state' => OrderInterface::STATE_CART],
+            ['state' => OrderCheckoutStates::STATE_CART],
```

Comment:
To be reverted.
Although the value is the same, the field we're using is `state`, not `checkout_state`